samples/google/sheets/program.py

```diff
@@ -5,52 +5,37 @@
 """
 
 import autokitteh
-from autokitteh.google import google_id, google_sheets_client
-from autokitteh.slack import slack_client
+from autokitteh.google import google_sheets_client
 
 
 sheet = google_sheets_client("sheets_conn").spreadsheets().values()
-slack = slack_client("slack_conn")
 
 
-def on_slack_slash_command(event):
-    """Use a Slack slash command to interact with a Google Sheet.
+def on_http_get(event):
+    """Entry point for the workflow.
 
-    See: https://api.slack.com/interactivity/slash-commands, and
-    https://api.slack.com/interactivity/handling#message_responses
+    This function expects the URL parameter 'id' to be:
+    - A valid Google Sheets ID (see https://developers.google.com/sheets/api/guides/concepts)
 
-    In this sample, we expect the slash command's text to be either:
-    - A Google Sheets ID (https://developers.google.com/sheets/api/guides/concepts)
-    - A full Google Sheets URL (to extract the spreadsheet ID from it)
+    Example URL: "http://localhost:9980/webhooks/<webhook-slug>?id=<Google-Sheets-ID>"
 
     Args:
-        event: Slack event data.
+        event: HTTP event data, including URL query parameters.
     """
-    user_id = event.data.user_id
-    sheet_id = _extract_sheet_id(event.data.text, user_id)
+    sheet_id = event.data.url.query.get("id")
     if not sheet_id:
         return
```

Comment:
Maybe also print a message that a sheet ID is required but not found?

Reply:
Done